### PR TITLE
feat(astro): Migration first steps

### PR DESCRIPTION
**Pull request by AlexandreGaubert**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dist
```
Comment (jd) [inline .eslintignore:1]: Probably not related to migratino, do a PR for this?

File: .mergify.yml [hunk 2]
```diff
@@ -1,103 +0,0 @@
-extends: .github
```
Comment (jd) [inline .mergify.yml:1]: lol?

File: README.md [hunk 3]
```diff
@@ -1,20 +1,70 @@
-# Mergify Docs
+# Astro Docs <picture><source media="(prefers-color-scheme: dark)" srcset="https://astro.build/assets/press/astro-icon-light.png"><source media="(prefers-color-scheme: light)" srcset="https://astro.build/assets/press/astro-icon-dark.png"><img align="right" valign="center" height="79" width="63" src="https://astro.build/assets/press/astro-icon-dark.png" alt="Astro logo" /></picture>
```
Comment (jd) [inline README.md:1]: what?

File: .node-version
Comment (sileht) [inline .node-version:1]: Can we keep that one, it's used by Cloudflare.

**Comment by AlexandreGaubert** [comment]
@jd @sileht This is not the final version, I am aware that I need to cut a lot of things, I was just attempting to get this running in CI and get the prod build preview online, to see if code is ok.

**Comment by jd** [comment]
I guessed so, I stopped reviewing soon 😆

**Comment by sileht** [comment]
Here, a couple of things that could be improved: 
CleanShot 2023-12-14 at 11 23 14

**Comment by AlexandreGaubert** [comment]
Lighthouse results for desktop:
image

And for mobile:
image

File: .gitignore [hunk 4]
```diff
@@ -1,8 +1,38 @@
...
 .env.production
+.env.development
+
+# IDE environments
+.vscode/
+.idea/
+.vimrc
+.nanorc
+.stackblitzrc
+sandbox.config.json
```
Comment (sileht) [inline .gitignore:29]: Unrelated to the project

File: .gitignore [hunk 5]
```diff
@@ -1,8 +1,38 @@
...
+package-lock.lock
+package-lock.json
+yarn.lock
+
+# logs
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+.pnpm-debug.log*
```
Comment (sileht) [inline .gitignore:16]: Unrelated to the project

File: .gitignore [hunk 6]
```diff
@@ -1,8 +1,38 @@
...
-.env.development
+# build output
+dist
+.astro/
+
+# dependencies
+node_modules/
+package-lock.lock
+package-lock.json
+yarn.lock
```
Comment (sileht) [inline .gitignore:9]: We don't use package manager that generate these files

File: .gitignore [hunk 7]
```diff
@@ -1,8 +1,38 @@
...
+sandbox.config.json
+
+# macOS-specific files
+.DS_Store
 
-# Statically generated images at build time, don't push them
-/static/og-images
\ No newline at end of file
+# Local Netlify folder
+.netlify
```
Comment (sileht) [inline .gitignore:35]: Unrelated

File: .prettierignore [hunk 8]
```diff
@@ -0,0 +1,7 @@
+dist
+node_modules
+.github
+.changeset
+*.md
+*.mdx
+pnpm-lock.yaml
```
Comment (sileht) [inline .prettierignore:7]: We don't use pnpm

**Comment by sileht** [comment]
@jd can you review the preview site: https://docs-preview.mergify.com/2938/docs/

**Comment by jd** [comment]
> @jd can you review the preview site: https://docs-preview.mergify.com/2938/docs/

https://www.loom.com/share/5e3665d0c2df439c9c349d7965e3bc11

**Comment by sileht** [comment]
There is still a lot of purple coming from default template everywhere:
CleanShot 2023-12-21 at 13 17 43@2x

**Comment by sileht** [comment]
CleanShot 2023-12-21 at 13 20 15@2x

**Comment by sileht** [comment]
Dark theme is full purple.

We need to preserve our color identity.

**Comment by jd** [comment]
@AlexandreGaubert https://socialsharepreview.com/?url=https://docs-preview.mergify.com/2938/docs/workflow/automerge/ seems broken now

**Comment by AlexandreGaubert** [comment]
> @AlexandreGaubert https://socialsharepreview.com/?url=https://docs-preview.mergify.com/2938/docs/workflow/automerge/ seems broken now

This is because `og:image` tags points to `docs.mergify.com`, but you can see them at `<env_url>/open-graph/<path_to_page>.png` example: https://fd628654.docs-8fr.pages.dev/open-graph/getting-started.png
